praise-participants.md

```diff
@@ -0,0 +1,60 @@
+## Title
+Praise Participants
+
+## Patlet
+After an inner source contribution it's important to thank the contributor for their time and effort.
+This pattern gives guidance for a way to do so that not only effectively acknowledges the contribution but also endgenders further engagement from the contributor and others.
+
+## Problem
+How can we properly express our gratitude to a contributor for their inner source contribution to a project?
+It can be easy to forget to do so or not know the words or medium to use for adequate effect and sincerity.
+A pattern in this area makes it easy to do and ensures that the message comes across clearly and sincerely.
+
+## Context
+* You are the trusted committer or maintainer on an inner source project.
```

Comment:
I also think it would be very important for manager of a project which is using an innersourced artifact to praise the project. I have seen up close what happens if that recognition is not given, especially when we're dealing with volunteers. I'm not sure this pattern is where this belongs, though - might be a pattern of its own, dealing with how to handle _voluntary_ contributions.

Reply:
This is praise on the other side, from the _user_ of a project to it's owners?